learning/tour-of-beam/frontend/lib/constants/sizes.dart

```diff
@@ -16,31 +16,37 @@
  * limitations under the License.
  */
 
-// spacings
-const double kZeroSpacing = 0;
-const double kSmSpacing = 4;
-const double kMdSpacing = 8;
-const double kLgSpacing = 12;
-const double kXlSpacing = 16;
-const double kXxlSpacing = 36;
+class ProjectSpacing {
+  static const double zero = 0;
+  static const double small = 4;
+  static const double medium = 8;
+  static const double large = 12;
+  static const double xl = 16;
+  static const double xxl = 36;
+}
 
-// sizes
-const double kFooterHeight = 32;
+class ProjectSizes {
+  static const double footerHeight = 32;
+}
 
-// border radius
-const double kSmBorderRadius = 4;
-const double kMdBorderRadius = 6;
-const double kLgBorderRadius = 8;
-const double kXlBorderRadius = 28;
+class ProjectBorderRadius {
+  static const double small = 4;
+  static const double medium = 6;
+  static const double large = 8;
+  static const double xl = 28;
+}
 
-// elevation
-const double kElevation = 2;
+class ProjectElevation {
```

Comment:
Let's only keep those that are used. This one is not.

Reply:
I've deleted redundant constants.